cit_041a_gtfs_point_locations/README.md

```diff
@@ -1,9 +1,13 @@
 ## Cities with Transit Feed Data Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Cities with GTFS dataset](https://openmobilitydata.org/) for [display on Resource Watch](https://resourcewatch.org/data/explore/cit041-Transit-Feed).
 
-{Describe how the original data came from the source.}
-
-{Describe the steps used to process the data, e.g., "convert variable X from the original netcdf file to a tif to upload to Google Earth Engine."}
+This dataset was provided by the source as a JSON file. The JSON file contains a list of all the locations for which transit feed data is available. The id feature from this JSON was used to access individual JSON files for each location that contain more attributes about them. Those JSONs were transformed into a table so that it could be uploaded to Carto. In order to transform the data from the JSON to the data table used by Resource Watch, the following changes were made:
+- A unique ID for each location was created using the 'id' variable of the 'results' feature from the primary JSON. This was stored in a new column called 'feed_id'.
+- A pandas dataframe was created using the 'feeds' variable from the 'results' feature of each secondary JSON.
```

Comment:
```suggestion
- A pandas dataframe was created using the 'feeds' variable from the 'results' feature of each secondary JSON. The nested structure of the JSON was unpacked so that each variable had its own column with only one value. 
```